src/back8sup.sh

```diff
@@ -32,6 +32,7 @@ readonly DST_FOLDER=${DST_FOLDER:-/mnt/back8sup}
 readonly CONFIGMAP_PATH=${CONFIGMAP_PATH:-/etc/config.yaml}
 readonly EXPORT_FORMAT=${EXPORT_FORMAT:-yaml}
 readonly BINARIES="kubectl yq jq yamllint"
+readonly NOTNAMESPACEDDIR=${NOTNAMESPACEDDIR:-not-namespaced}
```

Comment:
Did you consider defaulting this to `kubectl api-resources --namespaced=false`?

Reply:
this is the destination dir for the notnamespaced resources, why should it default to `kubectl api-resources --namespaced=false` ?